src/progress-indicator/progress-indicator.style.tsx

```diff
@@ -109,22 +52,24 @@ export const IndicatorBar = styled.div<IndicatorProps>`
             background-color: ${color(props)};
         `;
     }};
-
-    ${MediaQuery.MaxWidth.mobileL} {
-        width: ${INDICATOR_BAR_WIDTH_MOBILE + "rem"};
-    }
 `;
 
-export const IndicatorTitle = styled(Text.BodySmall)<IndicatorProps>`
-    margin-right: 0.5rem;
-
+export const IndicatorTitleDesktop = styled(Text.BodySmall)<IndicatorProps>`
+    overflow-wrap: anywhere;
     ${(props) => {
         const { highlighted } = props;
         const color = highlighted ? Color.Primary : Color.Neutral[4];
         return `color: ${color(props)};`;
     }};
 `;
 
+export const IndicatorTitleMobile = styled(Text.BodySmall)`
+    overflow-wrap: anywhere;
+    ${(props) => {
+        return `color: ${Color.Neutral[1](props)};`;
+    }};
```

Comment:
since this is constant, we can simplify this to `color: ${Color.Neutral[1]};` without needing to nest a function

Reply:
more of a code style issue but is functionally correct, will close for now